### ISSUE TITLE
Ability to send test emails

### ISSUE THREAD

**Issue by dorsegal** [Netflix/security_monkey#752]
Hi,

Im trying to setup SES/SMTP server for alerts.
I tihknk it will be nice to add the monkey a param for testing those configuration
e.g:
monkey send_email admin@example.com 
and it will send a test massage.

**Comment by monkeysecurity** [comment]
not a bad suggestion

**Comment by thatguyfig** [comment]
Agreed, maybe even configuration of mail groups via the UI would be handy. Then you can add users to these groups and add the group as a notification target.
